Projects/Data/Sources/Entities/Post.swift

```diff
@@ -0,0 +1,310 @@
+        case isOriginalContent = "is_original_content"
+        case userReports = "user_reports"
+        case secureMedia = "secure_media"
+        case isRedditMediaDomain = "is_reddit_media_domain"
+        case isMeta = "is_meta"
+        case category = "category"
+        case secureMediaEmbed = "secure_mediaEmbed"
+        case linkFlairText = "link_flair_text"
+        case canModPost = "can_mod_post"
+        case score = "score"
+        case approvedBy = "approved_by"
+        case thumbnail = "thumbnail"
+        case edited = "edited"
+        case authorFlairCSSClass = "author_flair_css_class"
+        case authorFlairRichText = "author_flair_richtext"
+        case gildings = "gildings"
+        case postHint = "post_hint"
+        case contentCategories = "content_categories"
+        case isSelf = "is_self"
+        case modNote = "mod_note"
+        case created = "created"
+        case linkFlairType = "link_flair_type"
+        case wls = "wls"
+        case bannedBy = "banned_by"
+        case authorFlairType = "author_flair_type"
+        case contestMode = "contest_mode"
+        case selfTextHTML = "selftext_html"
+        case likes = "likes"
+        case suggestedSort = "suggested_sort"
+        case bannedAtUtc = "banned_at_utc"
+        case viewCount = "view_count"
+        case archived = "archived"
+        case noFollow = "no_follow"
+        case isCrosspostable = "is_crosspostable"
+        case pinned = "pinned"
+        case over18 = "over_18"
+        case preview = "preview"
+        case mediaOnly = "media_only"
+        case linkFlairTemplateID = "link_flair_template_id"
+        case canGild = "can_gild"
+        case spoiler = "spoiler"
+        case locked = "locked"
+        case authorFlairText = "author_flair_text"
+        case visited = "visited"
+        case numReports = "num_reports"
+        case distinguished = "distinguished"
+        case subredditID = "subreddit_id"
+        case modReasonBy = "mod_reason_by"
+        case removalReason = "removal_reason"
+        case linkFlairBackgroundColor = "link_flair_background_color"
+        case id = "id"
+        case isRobotIndexable = "is_robot_indexable"
+        case reportReasons = "report_reasons"
+        case author = "author"
+        case numCrossposts = "num_crossposts"
+        case numComments = "num_comments"
+        case sendReplies = "send_replies"
+        case whitelistStatus = "whitelist_status"
+        case modReports = "mod_reports"
+        case authorPatreonFlair = "author_patreon_flair"
+        case authorFlairTextColor = "author_flair_text_color"
+        case permalink = "permalink"
+        case parentWhitelistStatus = "parent_whitelistStatus"
+        case stickied = "stickied"
+        case url = "url"
+        case subredditSubscribers = "subreddit_subscribers"
+        case createdUTC = "created_utc"
+        case media = "media"
+        case isVideo = "is_video"
```

Comment:
Great job implementing the models! 👍 
However, could you clarify where did you take those coding keys from? Was it retrieved from the `get` request that you did or from some documentation piece?
I'm asking because I wasn't able to find that list on official API documentation or by googling.

Reply:
Just reverse engineering, could not find all of the info public. According to some Reddit posts, not all of the fields are documented.